personalized_active_learning/datasets/base.py

```diff
@@ -404,22 +463,8 @@ def _split_data(self) -> None:
 
         """
         if self.split_mode == SplitMode.PREDEFINED:
-            if "split" not in self.annotations.columns:
-                raise Exception(
-                    "Split mode {0} is used but no column split in {1}".format(
-                        self.split_mode.value,
-                        self.annotations.columns,
-                    )
-                )
             return
         elif self.split_mode == SplitMode.TEXTS:
-            if "fold" not in self.annotations.columns:
```

Comment:
Why it is removed?

Reply:
Cause the check is done in line 273. I think that double check is not necessary